osh/builtin_comp.py

```diff
@@ -335,7 +358,7 @@ def __init__(self, comp_state, errfmt):
   def Run(self, cmd_val):
     # type: (cmd_value__Argv) -> int
     argv = cmd_val.argv[1:]
-    arg_r = args.Reader(argv)
+    arg_r = args.Reader(argv, [])
```

Comment:
I think the second arg here can be `cmd_val.arg_spids[1:]`

The idea there is to be able to "blame" arguments in errors.  The "spids" are location info that allow that (though they're also the thing I'm in the middle of refactoring!)